build/phpcs.xml

```diff
@@ -2,9 +2,13 @@
 <ruleset name="Slevomat Coding Standard">
 	<arg name="extensions" value="php"/>
 	<arg name="tab-width" value="4"/>
+	<arg name="cache" value="./../temp/phpcs"/>
 	<arg value="s"/>
 	<arg value="p"/>
 
+	<file>./../</file>
+	<exclude-pattern>*/vendor/*</exclude-pattern>
```

Comment:
The previous configuration looks different:

```
<arg value="--ignore=*/data/*"/>
<arg path="SlevomatCodingStandard"/>
<arg path="build/PHPStan"/>
<arg path="tests"/>
```

For me the whitelist is better.

Reply:
Ah, I don't know why it wasn't failing locally for me, but at least now understand the `/data/` exclusion.

I'll bring that back and then the build should pass.